src/applications/personalization/dashboard/utils/getStatusContents.jsx

```diff
@@ -53,7 +54,7 @@ export function getStatusContents(appeal, name = {}) {
       break;
     case STATUS_TYPES.pendingForm9: {
       const formattedSocDate = format(
-        addDays(new Date(details.lastSocDate), 1),
+        new Date(replaceDashesWithSlashes(details.lastSocDate)),
```

Comment:
looks like in each of these, we are getting rid of adding a day to the timestamp. Is this the intended behavior?

Reply:
right, i made these changes (adding `addDays`) a few months ago due to CI tests failing due to the [one-day-off issue with Javascript Date object ](https://meta.stackoverflow.com/questions/421831/temporary-policy-generative-ai-e-g-chatgpt-is-banned?cb=1) 😭 so replacing the dashes with slashes helped make the CI tests pass.